js/adapt-youtube.js

```diff
@@ -1,208 +1,206 @@
-define([
-  'core/js/adapt',
-  'core/js/views/componentView',
-  'core/js/models/componentModel'
-], function(Adapt, ComponentView, ComponentModel) {
-
-  var YouTubeView = ComponentView.extend({
-    player: null,
-
-    events: {
+import Adapt from 'core/js/adapt';
+import ComponentView from 'core/js/views/componentView';
+import ComponentModel from 'core/js/models/componentModel';
+
+class YouTubeView extends ComponentView {
+  
+  get template() {
+    return 'youtube';
+  }
+
+  events() {
+    return {
       'click .js-youtube-inline-transcript-toggle': 'onToggleInlineTranscript',
       'click .js-youtube-external-transcript-click': 'onExternalTranscriptClicked',
       'click .js-skip-to-transcript': 'onSkipToTranscript'
-    },
-
-    initialize: function() {
-      ComponentView.prototype.initialize.apply(this);
-
-      _.bindAll(this, 'onPlayerStateChange', 'onPlayerReady', 'onInview');
-
-      this.debouncedTriggerGlobalEvent =_.debounce(this.triggerGlobalEvent.bind(this), 1000);
+    };
+  }
+
+  initialize() {
+    super.initialize();
+
+    _.bindAll(this, 'onPlayerStateChange', 'onPlayerReady', 'onInview');
+
+    this.player = null;
+    this.debouncedTriggerGlobalEvent =_.debounce(this.triggerGlobalEvent.bind(this), 1000);
```

Comment:
ESLint: space needed.

```suggestion
    this.debouncedTriggerGlobalEvent = _.debounce(this.triggerGlobalEvent.bind(this), 1000);
```